node/static/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>MongoDB Charts Embedding Example (Node)</title>
+
+    <script>
+        function renderSignedAuthChart(chartId, iframeId) {
+            let embedUrlRequest = new Request(`/api/embeddedchart/${chartId}`);
+            fetch(embedUrlRequest)
+                .then(response => response.text())
+                .then(url => document.getElementById(iframeId).src = url);
```

Comment:
It's best practice to not use assignment in return statements as this can lead to ambiguity in the intent.
Consider re-writing this line to:

```javascript
.then(url => { document.getElementById(iframeId).src = url });
```